testing/framework/TestSCons.py

```diff
@@ -310,7 +310,7 @@ def detect(self, var, prog=None, ENV=None, norm=None):
             if v != prog:
                 return None
             result = env.WhereIs(prog)
-            if norm and os.sep != '/':
+            if result and norm and os.sep != '/':
```

Comment:
Can you add to the docstring for detect what the arguments do?

Reply:
Looks like Norm should default to False and is a boolean flag to say whether to normalize the path?

Reply:
didn't look at norm... will do so.  just added the check that if env.WhereIs didn't have a good result, then doing replace on it throws error
```
AttributeError: 'NoneType' object has no attribute 'replace'
```
I'll update docstring.